include/z64math.h

```diff
@@ -145,4 +145,7 @@ typedef VecSphGeo VecGeo;
 #define SQXYZ(vec) ((vec).x * (vec).x + (vec).y * (vec).y + (vec).z * (vec).z)
 #define DOTXYZ(vec1, vec2) ((vec1).x * (vec2).x + (vec1).y * (vec2).y + (vec1).z * (vec2).z)
 
+f32 Rand_ZeroFloat(f32 f);
+f32 Rand_CenteredFloat(f32 f);
```

Comment:
initial reaction is that this doesnt fit here in math.h, but I understand the dillema here. The functions live in z_actor and are used by more then actors. We probably cant put it with the other rand functions cause those are from an external library right

Reply:
yeah I have the same train of thoughts.
(note this location is copied from MM)

Maybe we could create a new rand.h and rename existing rand.h to librand.h ?

Reply:
possibly
I also remember some people wanting to use the original function names for the library functions.
A game-side header could give the opportunity for both options to coexist: The library uses the real names, and we give them a wrapper define on the game-side with a name we like

Reply:
#2164